app/jsx/components/TabMainComp.jsx

```diff
@@ -14,7 +14,6 @@ const config = {
   },
   tex2jax: {
     inlineMath: [
-      ["$", "$"],
```

Comment:
Do we still need this config in TabMainComp?

Reply:
Good catch, we do not.